vipps-login-api.md

```diff
@@ -614,11 +614,11 @@ an authorization code and delivers it to the client by adding the following
 parameters to the query component of the redirection URI using the
 `application/x-www-form-urlencoded` format.
 
-| Query             | Description                                                                                                                                                                                                   |
-| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| code              | The authorization code generated by the authorization server. The client MUST NOT use the authorization code more than once. The authorization code is bound to the client identifier and redirection URI.    |
-| state             | The exact value received from the client during the authorization request.                                                                                                                                    |
-| scope             | The scopes that the end user has consented to. This list will always be the same as in the request query as end users are not allowed to remove individual scopes when they give consent.                                                                                                                                        |
+| Query             | Description                                                                                                                                                                                                                                                            |
+| ----------------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| code              | The authorization code generated by the authorization server. This code should be sent with the subsequent `/token`request.  The client MUST NOT use the authorization code more than once. The authorization code is bound to the client identifier and redirect URI. |
```

Comment:
```suggestion
| code              | The authorization code generated by the authorization server. This code should be sent with the subsequent `/token`request.  The client **must not** use the authorization code more than once. The authorization code is bound to the client identifier and redirect URI. |
```

Reply:
The reason for having capital letters is to be consistent with [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119). We have tried to follow that consistently for the rest of the documentation